pages/index.tsx

```diff
@@ -23,6 +25,7 @@ interface ClassData {
 	course_id: string
 }
```

Comment:
(NOTE: repeated comment from [PR 110](https://github.com/omshub/website/pull/110), but including here as well for visibility / reference)

Let's start consolidating the data models so we're not doing double-work / independent definition. Here, let's use the corresponding interface defined in `firebase/documentDataTypes.ts` (i.e., `Course`); and similarly for other client-side stuff moving forward...

Also, let's generally switch the "class" semantics over to "course" as well (filenames, variables, etc.), since that's also kind of confusing.

Reply:
Resolved as of most recent `main`